Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -11411,6 +11421,16 @@ a_Player:OpenWindow(Window);
 					},
 					Notes = "Returns the player's maximum sprinting speed, relative to the game default speed. Defaults to 1.3, but plugins may modify it for faster or slower sprinting.",
 				},
+				GetSkinParts =
+				{
+					Returns =
+					{
+						{
+							Type = "number",
+						},
+					},
+					Notes = "Returns the player's current set of skin part flags.",
```

Comment:
A bit more info about the return value would be good - how is the set represented, what does the number mean, a link to the skin part flags enum...